examples/step-44/step-44.cc

```diff
@@ -31,10 +31,13 @@
 #include <deal.II/base/tensor.h>
 #include <deal.II/base/timer.h>
 #include <deal.II/base/work_stream.h>
-
 #include <deal.II/dofs/dof_renumbering.h>
 #include <deal.II/dofs/dof_tools.h>
 
+// This header gives us the functionality to store
+// data at quadrature points
+#include <deal.II/base/quadrature_point_data.h>
```

Comment:
Understood :-)
> On 25 May 2016, at 17:40, Wolfgang Bangerth <notifications@github.com> wrote:
> 
> In examples/step-44/step-44.cc <https://github.com/dealii/dealii/pull/2620#discussion_r64595613>:
> 
> >  #include <deal.II/dofs/dof_renumbering.h>
> >  #include <deal.II/dofs/dof_tools.h>
> >  
> > +// This header gives us the functionality to store
> > +// data at quadrature points
> > +#include <deal.II/base/quadrature_point_data.h>
> OK, makes sense. I just wanted it discussed and not "just happen".
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dealii/dealii/pull/2620/files/a8902e5afe3229dbc3a823d07445459a2d04d9b1#r64595613>